ingestion-edge/Dockerfile

```diff
@@ -1,11 +1,19 @@
-FROM python:3.7-stretch
+ARG PYTHON_VERSION=3.8
+
+# build requirements in separate stage because it requires gcc and libc-dev
+FROM python:${PYTHON_VERSION}-slim
 WORKDIR /app
-RUN apt-get update && apt-get install -y wrk
+RUN apt-get update && apt-get install -qqy gcc libc-dev
 COPY requirements.txt /app/
 COPY bin/include/common.sh /app/bin/include/
 COPY bin/build /app/bin/
 ENV VENV=false
 RUN bin/build
+
+FROM python:${PYTHON_VERSION}-slim
+WORKDIR /app
+COPY --from=relud/wrk:4.1.0-buster /usr/local/bin/wrk /usr/local/bin/wrk
```

Comment:
Please add a comment here that links to the source for `relud/wrk` and explains why this is needed.